pkg/operator/ceph/provisioner/controller/controller.go

```diff
@@ -767,7 +767,7 @@ func (ctrl *ProvisionController) provisionClaimOperation(claim *v1.PersistentVol
 		return nil
 	}
 
-	provisioner, parameters, err := ctrl.getStorageClassFields(claimClass)
+	provisioner, reclaimPolicyPtr, parameters, err := ctrl.getStorageClassFields(claimClass)
```

Comment:
Could you rename the `reclaimPolicyPtr` variable used here, the `Ptr` sounds misleading to old Kubernetes TPR feature.
Something like `reclaimPolicyField` should be good.